Assignment3/ques8.js

```diff
@@ -0,0 +1,33 @@
+// ? 8. Write a program to dispaly number from 1-20 using all types of loops
+
+function differentTypesOfLoops() {
+  for (let i = 1; i <= 20; i++) {
+    console.log(i);
+  }
+
+  let i = 1;
+  while (i <= 20) {
+    console.log(i);
+    i++;
+  }
+
+  let j = 1;
+  do {
+    console.log(j);
+    j++;
+  } while (j <= 20);
+
+  for (const i in 20) {
+    console.log(i);
+  }
+
+  for (const i in new Array(50).fill(1)) {
```

Comment:
this loop is not working as expected

Reply:
i have removed the for loop block which is not working.